libs/wingsdk/src/target-tf-aws/function.ts

```diff
@@ -235,6 +236,25 @@ export class Function extends cloud.Function implements IAwsFunction {
     this.addEnvironment("WING_FUNCTION_NAME", name);
   }
 
+  /**
+   * @internal
+   * @param handler IFunctionHandler
+   * @returns the function code lines as strings
+   */
+  protected _getCodeLines(handler: IFunctionHandler): string[] {
+    const inflightClient = handler._toInflight();
+    const lines = new Array<string>();
+
+    lines.push('"use strict";');
+    lines.push(`const AWSXRay = require('aws-xray-sdk');`);
+    lines.push(`AWSXRay.captureHTTPsGlobal(require('http'));`);
```

Comment:
Would this also capture `fetch` which we are using to implement the wing `http` module?

Reply:
hm, good point - https://github.com/aws/aws-xray-sdk-node/issues/531 - no, it's not. They apparently recommend to use some open telemetry sdk. Will check.

Reply:
looks interesting https://aws-otel.github.io/

Also, seems to be less intrusive and more generic. Will see if that's really just pluggable via ENV, layer and permissions.

Reply:
yes! is it possible to use OT for everything instead of directly xray? that would be a much better and general solution.